pages/Contact-Us/contact.html

```diff
@@ -96,7 +96,7 @@ <h1 class="heading">CONTACT-US</h1>
                             required="">
                         <label for="telephone">Mobile Number : </label>
 
-                        <input type="telephone" name="telephone" id="telephone" placeholder="+91 1234567890">
+                        <input type="number" name="telephone" id="telephone" placeholder="+91 1234567890">
```

Comment:
please replace "number" with "tel". see <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel">this </a>documentation.